cloudformation/membership-attribute-service.json

```diff
@@ -303,23 +278,17 @@
           "Fn::Base64": {
             "Fn::Join": ["", [
               "#!/bin/bash -ev\n",
+              {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://gu-membership-attribute-service-dist/set-env.sh .\n"]]},
```

Comment:
If we are using the `pip` version of `awscli`, we can probably remove the region lookups here and in the following 2 `aws s3` commands